logagg_collector/collector.py

```diff
@@ -130,14 +130,14 @@ def register_to_master(self):
         try:
             register = requests.get(url)
             register_result = json.loads(register.content.decode('utf-8'))
-            if register_result['success']:
-                if register_result['result']['success']:
-                    return register_result
-                else:
-                    err_msg = register_result['result']
-                    raise Exception(err_msg)
+            if not register_result.get('success'):
+                err_msg = register_result.get('message')
+                raise Exception(err_msg)
+
+            if register_result.get('result').get('success'):
```

Comment:
This will throw an error 
the result of `register_result.get('result')` would be `None` and you are doing a get of `None`